Lib/test/test_range.py

```diff
@@ -490,8 +484,10 @@ def test_empty(self):
         self.assertNotIn(-1, r)
         self.assertNotIn(1, r)
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
+    # TODO: RUSTPYTHON.
+    # NOTE: Test passes, fast iterators are required for this to not
+    #       take ~ 1m.
+    @unittest.skip
```

Comment:
As the comment suggests, this takes time. Since the test suite is already lengthy, I've skipped it until fast iterators for ranges are implemented. Should I un-skip?

Reply:
I think this skip is ok. We are skipping a few tests like this.